parcel_test.go

```diff
@@ -28,60 +24,88 @@ func getTestParcel() Parcel {
 	}
 }
 
-// TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
-	store := NewParcelStore(db)
-	parcel := getTestParcel()
+    db, err := sql.Open("sqlite", "tracker.db")
+    require.NoError(t, err)
+    defer db.Close()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+    store := NewParcelStore(db)
 
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+    parcel := getTestParcel()
 
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+    number, err := store.Add(parcel)
+    if err != nil {
+        require.NoError(t, err)
+    }
+    assert.NotEmpty(t, number)
+
+    addedParcel, err := store.Get(number)
+    if err != nil {
+        require.NoError(t, err)
+    }
+
+    assert.EqualValues(t, parcel.Client, addedParcel.Client)
+    assert.EqualValues(t, parcel.Status, addedParcel.Status)
+    assert.EqualValues(t, parcel.Address, addedParcel.Address)
+    assert.EqualValues(t, parcel.CreatedAt, addedParcel.CreatedAt)
+
+    err = store.Delete(number)
+    require.NoError(t, err)
+
+    _, err = store.Get(number)
+    require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```